wiki/docs/developer-resources/functional-spec.md

```diff
@@ -0,0 +1,218 @@
+
+![General settings](functional-spec-images/general-settings.png)
+
+By default the "global" setting value are inherited by all sattelites, but can be overriden per satellite by selecting the 'override' checkbox in the correct cell. That enables the UI element to change the default value, as the image above shows for the passcode setting for Satellite B. In the image above, all other global settings are inherited to both Sattelite A and B.
+
+Any setting changes made are applied immediately. The setting page can be closed using the same button as was used to open it. This button is available on all settings pages.
+
+The following settings pages and buttons are available:
+- general settings
+- view-specific settings (clock, weather, etc)
+- add view button
+- access control settings
+
+#### General settings
+The general settings tab provides settings that are not view specific. The following settings are provided. This is not an exhaustive list and will be expanded in the future. Also, this list is not necessary going to be shown in this order:
+
+|Setting name|Required yes/no|Type|Default value|Notes|
+|---|---|---|---|---|
+|**Passcode**|No|Whole Number|None|Passcode to access the settings page|
+|**Default view**|Yes|View|Clock|The home screen to show on the satellite|
+|**Timeout (s)**|Yes|20|Whole Number|Timeout in seconds after which the satellites switch back to its default view|
+|**Default mode**|Yes|Enum (normal, night)|Normal|Available options: normal, night|
+|**Show top bar**|Yes|Boolean|False|Show or hide bar at top of the screen to provide touch access to views, similar to top bar in HA dashboards that allows access to views|
+|**Default do not disturb state**|Yes|Boolean|False||
+|**Launch icons**|No|String|Empty string|Need to flesh this out!|
+|**Status icon size**|No|Whole number|?|Need to flesh this out!|
+|**Font style**|Yes|String|Roboto|Options: Roboto, neeed to provide dropdown of options|
+|**24-hour clock**|Yes|Boolean|Dependent on HA config||
+|**Assistant**|Yes|Assistant entity name|Default HA assistant||
+|**TTS**|Yes|HA entity name in `tts` domain|If only one TTS entity select that as default|The TTS entity to use for any system generated messages|
+|**Assist Prompt**|Yes|Enum(none, flashing bar, blur pop up)|Blur pop up|how the Assist prompt is visualized. Available options: none/flashing bar/blur pop up|
+|**Background image**|Yes|File name string|None|The background image|
+|**Rotate background**|Yes|Boolean|False|Indicates if backgrounds should be rotated. If true, further settings are required|
+|**Background rotation interval**|No|Whole number and time unit enum (second/minute/hour/day)|1 hour|Rotate background every XX time (optional, only applies if rotate background is on). This replaces the current blueprint XX minutes and XX hours settings. It allows the user to enter a value and select a time unit (seconds/minutes/hours/days).|
```

Comment:
Agreed.  I don't like the current /5 formatting but I was either unable to work out something better or did not have time to find a workaround.  I think it was a limitation and was told it would not be an easy road.  Doing it here is great.